front/app/containers/Admin/settings/customize/AllInput.tsx

```diff
@@ -24,37 +24,32 @@ import messages from '../messages';
 import { isNilOrError } from 'utils/helperUtils';
 
 interface Props {
+  navbarItemSetting: boolean | null;
```

Comment:
The problem here is that the `updateAllInputInNavbar` variable in `settings/customize/index` (which is the value that is being passed to this `navbarItemSetting` prop) has be initialized with `true` or `false`. This initial value should be whether the item is currently in the navbar, which we get from the `useNavbarItemEnabled('all_input')` hook call. We could put that hook in `settings/customize/index` and initialize it from there. The problem is that that is currently a class component. If you want that, I guess there's three options

1. I rewrite it to a class component. Downside: I will not be able to finish this stuff before Sebi goes on holiday which will be annoying because he wants to finish the migrations
2. You rewrite it to a class component. Downside: you have to do it :)
3. We do this later

Reply:
I understand the urgency for getting this out. Front-end changes are easier to make than back-end changes. If it works now and we address it later (but not much later), we can leave as is. 

That being said, it's pretty easy to make a GetNavbarItemEnabled if you already have the hook. See e.g. GetAdminPublication.

(PS: if we're making changes, I also think the variable names need to change, not just the way this works. I don't know if I deleted this comment, but navbarSetting (in my head) is an object, not a boolean value. and the update... names were initially also quite confusing (had to dive in the code to get what they meant).)

Reply:
If you want I can still change the variable names? What about `navbarItemSetting` -> `navbarItemEnabled` and `updateEventsInNavbar` -> `newEventsNavbarItemEnabled`?

Reply:
That sounds clearer to me already. :-)